infrastructure/main.tf

```diff
@@ -51,6 +51,7 @@ module "bulk-scan" {
     FLYWAY_USER                   = "${module.bulk-scan-db.user_name}"
     FLYWAY_PASSWORD               = "${module.bulk-scan-db.postgresql_password}"
     //
+    SCAN_INTERVAL                 = "30000"
```

Comment:
Currently, this is the delay between the end of the last job execution and the start of the next. It would be best to either rename it to `SCAN_DELAY` (or sth similar) or set `fixedRateString` in `@Scheduled` annotation.